### DISCUSSION TITLE
Building an EVM compatible Blockchain with Ignite

### DISCUSSION THREAD

**Discussion by QuenumGerald** [cosmos/community#38] [Chain Builders (Cosmos SDK)]
Hello everyone, 

I'm exploring the possibility of building an EVM-compatible blockchain using lgnite .

Is this something that's feasible, and if so, could anyone guide me on how to get started? 

Any resources, advice, or personal experiences would be greatly appreciated. 

Thank you!

**Comment by julienrbrt** [comment]
Hi! You should take a look at https://github.com/evmos/ethermint.
A few chains such as [Evmos](https://github.com/evmos/evmos) or [Cronos](https://github.com/crypto-org-chain/cronos/) are implementing this module.

**Reply by QuenumGerald** [reply]
Yes im using the ethermint repo . Evmos is great but not open source.
